diff --git a/node_modules/prebuildify/index.js b/node_modules/prebuildify/index.js
index ca9829a..ffbb682 100644
--- a/node_modules/prebuildify/index.js
+++ b/node_modules/prebuildify/index.js
@@ -230,7 +230,8 @@ function build (target, runtime, opts, cb) {
     var child = proc.spawn(opts.nodeGyp, args, {
       cwd: opts.cwd,
       env: opts.env,
-      stdio: opts.quiet ? 'ignore' : 'inherit'
+      stdio: opts.quiet ? 'ignore' : 'inherit',
+      shell: true
     })
 
     child.on('exit', function (code) {
